src/__tests__/server/server.test.tsx

```diff
@@ -15,7 +16,7 @@ test("createIncludeElement", () => {
   const elem = createIncludeElement(
     "fragmentID",
     { name: "Kévin" },
-    { attrs: { alt: `"'<&Alt>'"` } }
```

Comment:
IIRC special chars are there to test that encoding is done properly. The previous value probably needs to be restored.

Reply:
I think this not necessary anymore as the esi tag was generated as a encoded string which was then  parsed and  decoded during the creation of the tag with:
```
        <div
          dangerouslySetInnerHTML={{
            __html: server.createIncludeElement(
              fragmentID,
              this.props,
              this.esi
            )
```
Or am I missing something?

Reply:
It's probably not necessary to encode the value anymore, but keeping special chars in the test will prevent any future regressions.

Reply:
Ok  after looking into it, we still need to encode but it should be safe to only do it for the alt attribute as this is the only one that may contain a URL.
And doing it with [encodeURIComponent](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent) should do the trick
What do you think?

Unfortunately I'm out of time do it, I'll be back in September

Reply:
I reverted to the original behavior